context/app/api/vitessce.py

```diff
@@ -65,30 +59,61 @@
     "salmon_rnaseq_10x": {
         "base_conf": SCATTERPLOT,
         "files_conf": [
-            {"rel_path": f"{SCRNASEQ_BASE_PATH}.cells.json", "type": "CELLS", },
-            {"rel_path": f"{SCRNASEQ_BASE_PATH}.cell-sets.json", "type": "CELL-SETS", },
+            {"rel_path": f"{SCRNASEQ_BASE_PATH}.cells.json", "type": "CELLS"},
+            {"rel_path": f"{SCRNASEQ_BASE_PATH}.cell-sets.json", "type": "CELL-SETS"},
         ],
     },
     "codex_cytokit": {
         "base_conf": IMAGING,
-        "view": {"zoom": -1.5, "target": [600, 600, 0], },
+        "view": {"zoom": -1.5, "target": [600, 600, 0]},
         "files_conf": [
             # Hardcoded for now only one tile.
             {
-                "rel_path": Path(CODEX_TILE_PATH) / Path(f"#CODEX_TILE#.ome.tiff"),
+                "rel_path": str(Path(CODEX_TILE_PATH) / "#TILE#.ome.tiff"),
                 "type": "RASTER",
-            }
+            },
+            {
+                "rel_path": str(Path(CODDEX_SPRM_PATH) / "#TILE#.cells.json"),
+                "type": "CELLS",
+            },
+            {
+                "rel_path": str(Path(CODDEX_SPRM_PATH) / "#TILE#.cell-sets.json"),
+                "type": "CELL-SETS",
+            },
+            {
+                "rel_path": str(Path(CODDEX_SPRM_PATH) / "#TILE#.genes.json"),
+                "type": "GENES",
+            },
+            {
+                "rel_path": str(Path(CODDEX_SPRM_PATH) / "#TILE#.clusters.json"),
+                "type": "CLUSTERS",
+            },
         ],
     },
 }
 
 CODEX_ASSAY = "codex_cytokit"
 
 IMAGE_ASSAYS = [CODEX_ASSAY]
+TILED_ASSAYS = [CODEX_ASSAY]
+
+TILE_REGEX = ".*(R(\\d+)_X(\\d+)_Y(\\d+)).*"
```

Comment:
```suggestion
TILE_REGEX = r"R\d+_X\d+_Y\d+"
```
- You almost always want raw strings with regexes in python.
- Are you using the inner capturing sub expressions?
- Use `search` instead of `match`, if you're only interested in the capture.